go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -22,6 +22,92 @@ func deleteVIPsFromAllOVNBalancers(vips sets.String, name, namespace string) err
 	err := deleteVIPsFromNonIdlingOVNBalancers(vips, name, namespace)
 	if err != nil {
 		return errors.Wrapf(err, "Failed to delete vips from ovn balancers %s %s", name, namespace)
+		
+type lbEndpoints struct {
+	IPs  []string
+	Port int32
```

Comment:
nit: why not use uint16?

Reply:
ack highest TCP port number is 16 bits

Reply:
actually, the endpoint port is defined as int32 in the kapi https://github.com/kubernetes/api/blob/v0.20.5/core/v1/types.go#L4484  

Which is why it has to stay as such